### ISSUE TITLE
Internal errors when porting DynamicVb6FormConverter.

### ISSUE THREAD

**Issue by Mo-Lo** [WaynePhillipsEA/twinbasic#129]
**Describe the bug**
I'm playing whit tB along with RC6 framework. I got 160+ internal error no resolve line associated end unresolve line end point when trying to port the code found at [https://www.vbforums.com/showthread.php?847047-Writing-a-new-IDE-with-clean-straight-forward-code-(vbRichClient)&p=5168415&viewfull=1#post5168415]

**To Reproduce**
I have attached the project files and the original files from the vbforums link .

**Expected behavior**
Should be no errors.

**Screenshots**
![image](https://user-images.githubusercontent.com/83885514/117553033-1652d700-b01d-11eb-8d02-ec0203b0c496.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.9.2653]
 - RC6 [v158.0]
 - RC6Widgets [v94.0]
 
**Additional context**
[DynamicVb6FormConverter.zip](https://github.com/WaynePhillipsEA/twinbasic/files/6446548/DynamicVb6FormConverter.zip)

**Comment by WaynePhillipsEA** [comment]
Thanks for reporting..   This embedded If line is throwing everything off:

    If L > 0 Then If Entries(EntriesCount).Content(0) = 123 Then Entries(EntriesCount).EntryType = frxTextRTF

This line is also causing a problem:
    Strm.SetPosition -8, STRM_SeekFromCurPos

I'll get them fixed soon.

**Comment by WaynePhillipsEA** [comment]
The second error is fixed in v0.9.2699.  The first embedded-If parsing error will be fixed tomorrow.

**Comment by WaynePhillipsEA** [comment]
So this is now fixed in v0.9.2720.  Thanks for reporting it.

Some other things that need a mention...   

1)  you need to reference RC5 rather than RC6, as it doesn't work with RC6 (same problem with VB6 + RC6).

2) You need to do this at the top of Main():
`   Set Cairo = New_c.Cairo
`
3)  Your substitutes for Screen.TwipsPerPixel are not quite right.  Change them to this:
   ```
xFac = apiGetPPI(True) / 1440.0
   yFac = apiGetPPI(False) / 1440.0
```

**Comment by Mo-Lo** [comment]
Thank you very much. I glad if the exercice somewhat help you improve tB.
